library/src/skins/default/src/sass/modules/tool/rubrics/_rubrics.scss

```diff
@@ -764,6 +764,10 @@ div.collapse-toggle-buttons > button {
     padding: 0.5rem;
 }
 
+sakai-rubric-summary div.panel-group {
```

Comment:
panel-group is part of BS-3. If you have time, you could perhaps take a look at swapping those panel-title etc for bs-5 card classes? Basically, panel doesn't do anything. Maybe that's why the margin wasn't working.